src/freenet/pluginmanager/OfficialPlugins.java

```diff
@@ -262,6 +274,7 @@ private OfficialPluginDescription createOfficialPluginDescription() {
 			this.deprecated = deprecated;
 			this.experimental = experimental;
 			this.advanced = advanced;
+      this.unsupported = unsupported;
```

Comment:
It sort of is - all added lines in this follow the [style guide](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.2-block-indentation). Does it make sense to follow the guide regardless of surrounding whitespace for new lines and automatically reindent the codebase once all pull requests are either merged or closed?